client/pages/home.jsx

```diff
@@ -50,30 +63,106 @@ function TopNav(props) {
+              date={trip.departureDate}
+              />
+          );
+        })
+      }
+    </ul>
+  );
+}
+
+function TripEntry(trip) {
+  const { name, date } = trip;
+  const monthNames = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+  const dateConvert = new Date(date);
+  dateConvert.setDate(dateConvert.getDate() + 1);
+  let month = dateConvert.getMonth();
+  month = monthNames[month];
+  const year = dateConvert.getFullYear();
+
   return (
-    <main className="d-flex flex-column justify-content-between">
-      <div>
-        <p className="h6 m-0 mt-3">Hello, <strong>username</strong></p>
-        <p className="m-0 p-0 mt-2 small text-muted"><span>What are we planning today?</span></p>
-        <hr className="dmx-2 my-4 d-block border-0" />
+    <li>
+      <div className="trip-entry d-flex border rounded-lg shadow-sm py-4 px-4 mb-2">
+        <a href="#" className="d-flex align-items-center">
+          <div className="shadow-sm rounded-lg p-2 border">
+            <svg xmlns="http://www.w3.org/2000/svg" width="2rem" height="2rem" fill="#FFAD0F" className="bi bi-pencil-square" viewBox="0 0 16 16">
+              <path d="M15.502 1.94a.5.5 0 0 1 0 .706L14.459 3.69l-2-2L13.502.646a.5.5 0 0 1 .707 0l1.293 1.293zm-1.75 2.456l-2-2L4.939 9.21a.5.5 0 0 0-.121.196l-.805 2.414a.25.25 0 0 0 .316.316l2.414-.805a.5.5 0 0 0 .196-.12l6.813-6.814z" />
+              <path fillRule="evenodd" d="M1 13.5A1.5 1.5 0 0 0 2.5 15h11a1.5 1.5 0 0 0 1.5-1.5v-6a.5.5 0 0 0-1 0v6a.5.5 0 0 1-.5.5h-11a.5.5 0 0 1-.5-.5v-11a.5.5 0 0 1 .5-.5H9a.5.5 0 0 0 0-1H2.5A1.5 1.5 0 0 0 1 2.5v11z" />
+            </svg>
+        </div></a>
+        <div className="d-flex flex-column justify-content-center flex-grow-1">
+          <h4 className="ml-4 m-0">{name}</h4>
+          <p className="text-muted small pt-1 ml-4 m-0">{`${month} ${year}`}</p>
+        </div>
+        <div className="d-flex align-items-center">
+          <a href="#">
+            <svg xmlns="http://www.w3.org/2000/svg" width="2rem" height="2rem" fill="#dbdbdb" className="icon bi bi-dash" viewBox="0 0 16 16">
+              <path d="M4 8a.5.5 0 0 1 .5-.5h7a.5.5 0 0 1 0 1h-7A.5.5 0 0 1 4 8z" />
+            </svg>
+          </a>
+        </div>
       </div>
-
-      <div className="light-teal text-center"><h3>Looks like you don&#39;t<br />have any trips planned!</h3></div>
-      <div className="light-teal text-center"><p>Let&#39;s start planning!<br />Click the <strong>ADD</strong> button to add a trip.</p></div>
-    </main>
+    </li>
   );
 }
 
 function BottomNav(props) {
   return (
     <footer className="container-xl footer d-flex justify-content-center align-items-center fixed-bottom px-1 w-100">
       <div className="text-center">
-        <div className="icon bg-white rounded-circle" onClick={props.onClick}>
+        <a href="#"><div className="icon bg-white rounded-circle" onClick={props.onClick}>
```

Comment:
🤔 Does this need to be wrapped in an anchor? We usually reserve those for links or hash route changes.

Reply:
I guess it doesn't, I wanted to make it so that the div (in this case the circle container) to be clickable.

Reply:
Found a way around that. I made it into a button. 🙃